src/main/java/com/gradle/enterprise/conventions/GradleEnterpriseConventionsPlugin.java

```diff
@@ -53,12 +53,18 @@ public void apply(Settings settings) {
             if (settings.getGradle().getStartParameter().isBuildCacheEnabled()) {
                 settings.buildCache(new BuildCacheConfigureAction(conventions));
             }
-            if (!settings.getGradle().getStartParameter().isNoBuildScan()) {
+            if (!settings.getGradle().getStartParameter().isNoBuildScan() && !containsPropertiesTask(settings)) {
                 configureBuildScan(settings, conventions);
             }
         });
     }
 
+    // Disable build scan for security reason
+    // https://github.com/gradle/gradle-enterprise-conventions-plugin/issues/9
+    private boolean containsPropertiesTask(Settings settings) {
+        return settings.getGradle().getStartParameter().getTaskNames().contains("properties");
```

Comment:
You can actually call `properties` on any project, so I think we should check if the task names contain `properties` or any task name ends with `:properties`.

Reply:
👍